fe/fe-core/src/main/java/org/apache/doris/alter/Alter.java

```diff
@@ -230,16 +230,6 @@ public void processAlterTable(AlterTableStmt stmt) throws UserException {
         List<AlterClause> alterClauses = Lists.newArrayList();
         // some operations will take long time to process, need to be done outside the table lock
         boolean needProcessOutsideTableLock = false;
-
-        // check conflict alter ops first
```

Comment:
Why do we needn't check conflict alter ops first?

Reply:
> Why do we needn't check conflict alter ops first?

alterClauses is always empty here, so check conflict alter ops makes no sense here.  in processAlterOlapTable and processAlterExternalTable will check conflict alter ops

Reply:
Ok, Thanks.